src/content/docs/journey-large-logs/filter.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Filter log ingest
+metaDescription: "Test"
+---
+
+import logsIngestPipeline from 'images/logs_diagram_ingest-pipeline.webp'
+
+import logsDropFilterSelection1 from 'images/logs_screenshot-full_drop-filter-selection-1.webp'
+
+import logsAttribute from 'images/logs_screenshot-full_drop-filter-attribute.webp'
+
+import logsDrop from 'images/logs_screenshot-drop-log.webp'
+
+import logsIntro from 'images/logs_screenshot-full_intro.webp'
+
+
+As we've talked about before, modern systems create massive amounts of logs. Not all of those are useful. In fact, there's a high chance when you look at your logs you'd find *most* aren't useful.
+
+
+<SideBySide>
+    <Side>
+
+New Relic provides the ability to create rule sets that look at your logs and ignore logs that you haven't selected for ingest. This has a few key benefits:
+
+
+* Lower costs by storing only the logs relevant to your account.
+* Lower costs by dropping specific attributes 
+* Reduce toil by only storing valuable logs.
+
+    </Side>
+    <Side>
+
+<img
+  title="Logs architecture for drop filters in New Relic"
+  alt="Diagram of logs architecture for drop filters in New Relic"
+  src={logsIntro}
+/>
+
+    </Side>
+</SideBySide>
+
+
+## How drop filter rules work [#how-it-works]
+
+A drop filter rule matches data based on a query. When triggered, the drop filter rule removes the matching data from the ingestion pipeline before it is written to the New Relic database (NRDB).
+
+This creates an explicit demarcation between the logs being forwarded from your domain and the data that New Relic collects. Since the data removed by the drop filter rule doesn't reach our backend, it cannot be queried: the data is gone and cannot be restored.
+
+## Decide which logs to drop [#decide]
+
+Deciding which logs to keep and which logs to drop is a highly specific decision for each team and organization. Logs valuable to one organization may not be valuable to another. Regardless, here are a few suggestions on how to decide which logs are valuable and which to drop:
+
+* **What logs do your team already use**: If your team already manually reviews a subset of logs regularly, that indicates those logs are valuable and should not be dropped. Likewise, if there is a set of lgos your team never looks at that might indicate they should be dropped.
+* **What apps and systems are producing the most logs**: an app or system that creates a large amount of logs indicates you should spend time deciding what to do with those logs. Is it a valuable and widely used app which indicates you should keep most of the logs? Is it a redundent system which is spewing logs with minimal value? 
```

Comment:
`What apps and systems are producing the most log` Alos feels like it can be more direct; I don't think you need `are producing` for instance vs `produce`